lib/matplotlib/tests/test_subplots.py

```diff
@@ -111,7 +111,7 @@ def test_exceptions():
     # the point of this test is to ensure that this raises.
     with warnings.catch_warnings():
         warnings.filterwarnings('ignore',
-                                message='.*sharex\ argument\ to\ subplots',
+                                message=r'.*sharex\ argument\ to\ subplots',
```

Comment:
What is `\ `? I don't think this escape even means anything in a regular expression.